src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -1930,6 +1930,29 @@ public ErrorDiagnostic RuntimeValueNotAllowedInFunctionDeclaration(string? acces
                 TextSpan,
                 "BCP343",
                 $@"Using a func declaration statement requires enabling EXPERIMENTAL feature ""{nameof(ExperimentalFeaturesEnabled.UserDefinedFunctions)}"".");
+        
+            public ErrorDiagnostic ExpectBodyStart() => new(
+                TextSpan,
+                "BCP345",
+                "Expected the \"{\" character at this location.");
+            public ErrorDiagnostic TestDeclarationMustReferenceBicepTest() => new(
+                TextSpan,
+                "BCP346",
+                "A Test declaration can only reference a Bicep File, an ARM template, a registry reference or a template spec reference.");
+
+            public ErrorDiagnostic ExpectedTestIdentifier() => new(
+                TextSpan,
+                "BCP0347",
+                "Expected a Test identifier at this location.");
```

Comment:
```suggestion
                "Expected a test identifier at this location.");
```

Super nitpicky of me, but please use standard sentence casing in error messages.